deployment-template/ecs/aws-otel-ec2-sidecar-deployment-cfn.yaml

```diff
@@ -65,6 +65,12 @@ Resources:
               awslogs-group: /ecs/aws-collector
               awslogs-region: !Ref 'AWS::Region'
               awslogs-stream-prefix: ecs
+        - healthCheck:
```

Comment:
typo? healthCheck vs HealthCheck?

Reply:
I will need to check that after running this cfn template - to be sure. So I will get back to you shortly on this.

Reply:
No need to check. It is in the documentation:
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-taskdefinition-containerdefinitions.html#cfn-ecs-taskdefinition-containerdefinition-healthcheck

Reply:
As we discussed, I have reverted the change I made to this file. This file needs to be handled in separate PR, as the original file is giving error upon deployment.